HdmiCecSink/HdmiCecSink.cpp

```diff
@@ -423,6 +425,7 @@ namespace WPEFramework
 		   registerMethod(HDMICECSINK_METHOD_REQUEST_ACTIVE_SOURCE, &HdmiCecSink::requestActiveSourceWrapper, this);
 		   registerMethod(HDMICECSINK_METHOD_SET_MENU_LANGUAGE, &HdmiCecSink::setMenuLanguageWrapper, this);
                    registerMethod(HDMICECSINK_METHOD_SETUP_ARC, &HdmiCecSink::setArcEnableDisableWrapper, this);
+		   registerMethod(HDMICECSINK_METHOD_DEBUG_ENABLE, &HdmiCecSink::setDebugEnableWrapper, this);
```

Comment:
We should not add client facing API to enable additional logging. We can look at increasing logging at runtime through the trace control plugin. Changes are planned in https://ccp.sys.comcast.net/browse/DELIA-48811.